sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/FhirIOTest.java

```diff
@@ -67,7 +68,8 @@ public void test_FhirIO_failedSearches() {
 
     PCollection<HealthcareIOError<String>> failed = searchResult.getFailedSearches();
 
-    PCollection<String> resources = searchResult.getResources();
+    PCollection<JsonArray> resources = searchResult.getResources();
+    resources.setCoder(JsonArrayCoder.of());
```

Comment:
I recommend you set the coder from inside your `expand` function in `FhirIO.java` - that way your users will not need to set their coders manually every time.

Reply:
Thanks and done!